internal/spokes/spokes.go

```diff
@@ -810,6 +810,10 @@ func (r *spokesReceivePack) readPack(ctx context.Context, commands []command, ca
 
 	if r.isFsckConfigEnabled() {
 		args = append(args, "--strict")
+		strictArgs := r.config.GetPrefix("receive.fsck")
```

Comment:
I think you should query here for `receive.fsck.` (note the final dot), otherwise, you will get back `.missingEmail=ignore`, won't you?

Reply:
Good catch!